app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/presenter/LearningPresenter.kt

```diff
@@ -1,16 +1,34 @@
 package com.github.HumanLearning2021.HumanLearningApp.presenter
 
 import android.app.Activity
+import android.net.Uri
 import android.util.Log
 import android.widget.ImageView
-import com.github.HumanLearning2021.HumanLearningApp.model.Category
-import com.github.HumanLearning2021.HumanLearningApp.model.DatabaseService
+import com.github.HumanLearning2021.HumanLearningApp.R
+import com.github.HumanLearning2021.HumanLearningApp.model.*
+import com.github.HumanLearning2021.HumanLearningApp.view.LearningMode
+import kotlinx.coroutines.runBlocking
+import java.lang.Exception
+import java.lang.IllegalArgumentException
 
-class LearningPresenter(
-    private val databaseService: DatabaseService
+
+class DummyLearningPresenter(
+    private val learningMode: LearningMode
 ) {
     private var previousCategory : Category? = null
+    private val databaseService = DummyDatabaseService()
```

Comment:
Are we not supposed to use DummyDBManagement? (his companion object to be exact) instead of directly using DummyDBService.

Reply:
Yes please do this @mlenw

Reply:
done